### PR TITLE
feat(database): add basic animation for database header drag handle

### PR DESCRIPTION
**Pull request by CatsJuice**

### CODE REVIEW

File: packages/blocks/src/database-block/table/table-view.ts [hunk 1]
```diff
@@ -54,6 +54,17 @@ const styles = css`
...
+    padding-top: 1.5px;
+  }
+  .affine-database-block-table::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    height: 1.5px;
+    background-color: var(--affine-border-color);
```
Comment (zzj3720) [inline packages/blocks/src/database-block/table/table-view.ts:67]: @zanwei Is this as expected?

Comment (zzj3720) [reply packages/blocks/src/database-block/table/table-view.ts:67]: 1.5px border at top of table view

Comment (zanwei) [reply packages/blocks/src/database-block/table/table-view.ts:67]: Based on what we discussed earlier, it should be 1px border at top of table view

Comment (CatsJuice) [reply packages/blocks/src/database-block/table/table-view.ts:67]: Got it!

**Review by zzj3720 — APPROVED** [review]

**Pull request merged**